### PR TITLE
made changes for enableing tagliatelle

### PR DESCRIPTION
**Pull request by vishwas-sharma2480**
This PR aims to enable the tagliatelle linter rule for our project. The tagliatelle rule is crucial for maintaining code quality and adherence to coding standards. The suggested solution involves evaluating whether we can afford to enable the rule. If affirmative, all linter errors have been addressed to ensure that the tests pass successfully. In case enabling the rule is deemed unfeasible, it has been disabled, and a comment is left to explain the decision (subject to review by the development team).

Changes Made:

Enabled the tagliatelle linter rule.
Addressed all linter errors to ensure successful test runs

### CODE REVIEW

File: managed/services/backup/pitr_timerange_service.go [hunk 1]
```diff
@@ -54,8 +54,8 @@ type oplogChunk struct {
 	RS          string              `bson:"rs"`
 	FName       string              `bson:"fname"`
 	Compression compressionType     `bson:"compression"`
-	StartTS     primitive.Timestamp `bson:"start_ts"`
-	EndTS       primitive.Timestamp `bson:"end_ts"`
+	StartTS     primitive.Timestamp `bson:"startTs"`
+	EndTS       primitive.Timestamp `bson:"endTs"`
```
Comment (BupycHuk) [inline managed/services/backup/pitr_timerange_service.go:58]: I think we need to revert it and comment to ignore it since it's based on data that we get from MongoDB.

**Comment by JiriCtvrtka** [comment]
@vishwas-sharma2480 Hi, thanks for you contribution. Could you run "make format" and commit changes to make main check pass?

**Comment by vishwas-sharma2480** [comment]
I have done the suggested changes @BupycHuk and @JiriCtvrtka let mw know if need any changes

File: managed/services/backup/pitr_timerange_service.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ type oplogChunk struct {
 	RS          string              `bson:"rs"`
 	FName       string              `bson:"fname"`
 	Compression compressionType     `bson:"compression"`
-	StartTS     primitive.Timestamp `bson:"start_ts"`
+	StartTS     primitive.Timestamp `bson:"start_Ts"`
```
Comment (BupycHuk) [inline managed/services/backup/pitr_timerange_service.go:57]: please revert this change

Comment (vishwas-sharma2480) [reply managed/services/backup/pitr_timerange_service.go]: Hi @BupycHuk I have done the requested changes

**Review by BupycHuk — APPROVED** [review]

**Review by JiriCtvrtka — APPROVED** [review]

**Comment by BupycHuk** [comment]
@vishwas-sharma2480 thank you for your contribution
